e107_plugins/news/templates/news_menu_template.php

```diff
@@ -130,6 +130,22 @@
 
 
 // TODO
-$NEWS_MENU_TEMPLATE['archive'] = array();
-
+$NEWS_MENU_TEMPLATE['archive']['start']       = '<ul class="news-archive-menu">';
+$NEWS_MENU_TEMPLATE['archive']['end']         = '</ul>';
+
+$NEWS_MENU_TEMPLATE['archive']['year-start']        = "<li>

```

Comment:
Great job on this. For consistency, this should use 'year_start' etc. ie. underscores instead of hyphens.